### ISSUE TITLE
CertPathValidatorException while generating session

### ISSUE THREAD

**Issue by prasadraju070** [angelbroking-github/smartapi-java#17]
E:\PROJECTS\Angel_Broking\TradingBot>java tradingbot.TradingBot
PKIX path validation failed: java.security.cert.CertPathValidatorException: validity check failed
exception createPostRequest
null
Parameter specified as non-null is null: method okhttp3.OkHttpClient.newCall, parameter request

kindly help me to resolve the issue

**Comment by techpvk** [comment]
Yes,even i faced same issue

**Comment by garry07396** [comment]
Yes facing this issue just today. Please help @kadam-mithun @vrushalikapse @angelbroking-github

**Comment by lokeshgidra** [comment]
I'm also facing the exact same problem. Started today only.

**Comment by CAPawanJangid** [comment]
Yes facing the same issue..

**Comment by prasadraju070** [comment]
I think their SSL certificate for their domain expired   ... They should
renew it

**Comment by garry07396** [comment]
@prasadraju070  admin replied in smart api forum, they are checking
